Ticket - 28691 ALL PACKAGING - Printing POs.

sewa67 opened the pull request:
Please review this change and merge into development branch
Files - po/po-lovetencan.i, po/po-print.i, po/po-xprnt10.p, po/po-xprt20.p, porep/r-poprt.w

sewa67 committed:
Ticket - 28691 ALL PACKAGING - Printing POs.
Files - po/po-lovetencan.i, po/po-print.i, po/po-xprnt10.p, po/po-xprt20.p, porep/r-poprt.w

brad-advantzware reviewed (changes requested):
Bad variable name.  cCustCode must be lCustCode.
cCustCode implies a char variable rather than logical.

sewa67 committed:
Ticket - 28691 ALL PACKAGING - Printing POs.
Files -  po/po-lovetencan.i, po/po-print.i, po/po-xprnt10.p, po/po-xprt20.p, porep/r-poprt.w
